README.md

```diff
@@ -436,11 +436,13 @@ Good starting values:
 
 This list lets you set alternative time (in ticks) to despawn certain types of dropped items faster or slower than default. This option can be used instead of item clearing plugins along with `merge-radius` to improve performance.
 
-#### use-faster-eigencraft-redstone
+#### redstone-implementation
 
-`Good starting value: true`
+`Good starting value: alternate-current`
+
+When enabled, the redstone system is replaced by a faster and alternative version that reduces redundant block updates, lowering the amount of work your server has to do. Enabling this can significantly improve performance without introducing gameplay inconsistencies. This option will additionally fix some redstone inconsistencies from CraftBukkit.
```

Comment:
> without introducing gameplay inconsistencies

is that stated and confirmed somewhere? I know eigencraft was at least pretty close to no differences, haven't looked into alternate current just yet.

Reply:
I believe AC has about the same amount of inconsistencies as eigencraft, but I'm not certain

Reply:
Yeah, let's rephrase that so it's clear it's not 100% the same, but waay faster

Reply:
If it actually breaks anything (referring to "inconsistencies"), I wouldn't set it as a default. If you don't know exactly what can be broken with this option at least. If you can find somewhere that lists these changes, it would probably be helpful to add a link to that too.